apps/devhub/tests/test_views_versions.py

```diff
@@ -467,10 +467,15 @@ def test_should_accept_zip_source_file(self):
             source_file.seek(0)
             data = self.formset(source=source_file)
             response = self.client.post(self.url, data)
-            eq_(response.status_code, 302)
-            version = Version.objects.get(pk=self.version.pk)
-            assert version.source
-            assert version.addon.admin_review
+        eq_(response.status_code, 302)
+        version = Version.objects.get(pk=self.version.pk)
+        assert version.source
+        assert version.addon.admin_review
+
+        log = ActivityLog.objects.get(action=amo.LOG.REQUEST_SUPER_REVIEW.id)
+        assert log.details['comments'] == (
```

Comment:
We've switched from `django_nose` to `pytest`, which makes good use of `assert` by introspecting them and providing a very useful failure message. Much more pythonic ;)